sdk/keyvault/keyvault-keys/README.md

```diff
@@ -207,6 +207,113 @@ const updatedKey = await client.updateKey(keyName, result.version, { enabled: fa
 await client.deleteKey(keyName);
 ```
 
+## Cryptography
+
+Additionally, a set of cryptography functions that are available through the `CryptographyClient`. Similar to the `KeysClient`, `CryptographyClient` will connect to Azure Key Vault with the provided set of credentials. Once connected, `CryptographyClient` can encrypt, decrypt, sign, verify, wrap keys, and unwrap keys.
+
+### Authenticate the client
+
+```typescript
+import { EnvironmentCredential } from "@azure/identity";
+import { KeysClient, CryptographyClient } from "@azure/keyvault-keys";
+```
+
+Once these are imported, we can next connect to the key vault service. To do this, we'll need to copy some settings from the key vault we are connecting to into our environment variables. Once they are in our environment, we can access them with the following code:
+
+```typescript
+// EnvironmentCredential expects the following three environment variables:
+// * AZURE_TENANT_ID: The tenant ID in Azure Active Directory
+// * AZURE_CLIENT_ID: The application (client) ID registered in the AAD tenant
+// * AZURE_CLIENT_SECRET: The client secret for the registered application
+const credential = new EnvironmentCredential();
+
+// Build the URL to reach your key vault
+const vaultName = "<YOUR KEYVAULT NAME>";
+const url = `https://${vaultName}.vault.azure.net`;
+
+// Connect to the key vault service
+const keysClient = new KeysClient(url, credential);
+
+// Create or retrieve a key from the keyvault
+let myKey = await client.createKey("MyKey", "RSA");
+
+// Lastly, create our cryptography client and connect to the service
+// This example uses the URL that is part of the key we created (called key ID or kid)
+const cryptographyClient = new CryptographyClient(url, myKey.keyMaterial!.kid!, credential);
+```
+
+### Encrypt
+`encrypt` will encrypt a message.
```

Comment:
Should we talk about the supported algorithms? It might be useful to point back to the service documentation too. Namely:
- https://docs.microsoft.com/en-us/rest/api/keyvault/encrypt/encrypt#jsonwebkeyencryptionalgorithm
- https://docs.microsoft.com/en-us/rest/api/keyvault/sign/sign#jsonwebkeysignaturealgorithm

Reply:
I'm not sure! I think they might change and maybe better for them to find out through intellisense? Hmm

Reply:
Added